api/tests/unit/domain/events/handle-certification-scoring_test.js

```diff
@@ -202,6 +221,12 @@ describe('Unit | Domain | Events | handle-certification-scoring', function () {
           .withArgs(certificationAssessment.certificationCourseId)
           .resolves(certificationCourse);
         certificationCourseRepository.update.resolves(certificationCourse);
+        complementaryCertificationCourseRepository.hasComplementaryCertification
```

Comment:
Vu qu'on est dans un contete 'when scoring is successful', je ne comprends pas pourquoi le repo retourne false

Reply:
En fait c'est un loupé, ce repo n'est pas utilisé ici, ca n'a pas d'impact (mais il faut le degager)